### DISCUSSION TITLE
unavailability of videos of people programming in java on youtube

### DISCUSSION THREAD

**Discussion by GrimReaper3223** [community/community#65188] [Programming Help]
### Select Topic Area

General

### Body

I believe that people like me (beginner in java), have already searched YouTube and other social networks for videos of other people programming in java, so that we can analyze the way that someone else programs so that we can absorb certain experiences of how to assemble a class or structure of a project

I don't know why there aren't so many people programming in java anymore (neither by video, nor by projects here at Git). It's a wonderful language, easy to learn and to set up a project, it's like playing tetris.

hopefully this will change in the future...

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GrimReaper3223 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by dushmanta05** [comment]
I'm currently learning Java, and I don't get your question. Can you further explain? Are you saying there are not enough resources or not enough project-oriented resources to study?

**Reply by GrimReaper3223** [reply]
sorry for the lack of clarity. what I meant was that we learned a lot of content about java (and other languages too that people don't like to make a lot of videos showing their programming style - it can be an ASMR or not-...), and there comes a certain moment when we have many codes in our mind, but sometimes we don't know how to start using that learning (at least with me and so...)

I learn java not through a course, but through the dev.java website. So I go to youtube to watch videos of other people programming in java (whether it's an app or something, just to see how to start assembling a file and also to know how to use what we've learned), and there aren't many videos that way .

**Reply by dushmanta05** [reply]
I guess it's mostly because Java is being used with frameworks like _Spring_ and _Spring Boot_, although there are beginner level projects on YouTube I think.

**Comment by GrimReaper3223** [comment]
could it be. The Spring framework has been very popular lately.
